### PR TITLE
Days since installation strings (EXPOSRUEAPP-5170)

### PR DESCRIPTION
**Pull request by SamuraiKek**
Strings for the new days since installation.

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values-bg/strings.xml [hunk 1]
```diff
@@ -344,8 +344,8 @@
     <string name="risk_details_subtitle_period_logged">"Този период се включва в изчислението."</string>
     <!-- XHED: risk details - infection period logged information body, below behaviors -->
     <string name="risk_details_information_body_period_logged">"Вашият риск от заразяване може да се изчисли само за периодите, в които регистрирането на излагания на риск е било активно. Затова тази функция трябва да бъде активирана постоянно."</string>
-    <!-- XHED: risk details - infection period logged information body, below behaviors -->
-    <string name="risk_details_information_body_period_logged_assessment">"Регистрирането на излагания на риск покрива последните 14 дни. За този период от време функцията е била активна на Вашия смартфон в продължение на %1$s дни. Приложението изтрива автоматично по-старите регистри, тъй като те вече не могат да служат за предотвратяване на заразяването."</string>
+    <!-- XTXT: risk details - infection period logged information body, under 14 days -->
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/res/values-bg/strings.xml:347]: Please don't change the strings and comments, only change it in german and defaults. All other languages will be covered by the transition system. The string will just be unused until it's replaced.

File: Corona-Warn-App/src/main/res/values-en/strings.xml [hunk 2]
```diff
@@ -344,8 +344,8 @@
     <string name="risk_details_subtitle_period_logged">"This period is included in the calculation."</string>
     <!-- XHED: risk details - infection period logged information body, below behaviors -->
     <string name="risk_details_information_body_period_logged">"Your risk of infection can be calculated only for periods during which exposure logging was active. The logging feature should therefore remain active permanently."</string>
-    <!-- XHED: risk details - infection period logged information body, below behaviors -->
-    <string name="risk_details_information_body_period_logged_assessment">"Exposure logging covers the past 14 days. During this time, the logging feature on your smartphone was active for %1$s days. The app automatically deletes older logs, as these are no longer relevant for infection prevention."</string>
+    <!-- XTXT: risk details - infection period logged information body, under 14 days -->
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/res/values-en/strings.xml:347]: same as above

File: Corona-Warn-App/src/main/res/values-pl/strings.xml [hunk 3]
```diff
@@ -344,8 +344,8 @@
     <string name="risk_details_subtitle_period_logged">"Ten okres jest uwzględniony w obliczeniu."</string>
     <!-- XHED: risk details - infection period logged information body, below behaviors -->
     <string name="risk_details_information_body_period_logged">"Ryzyko zakażenia można obliczyć tylko dla okresów, w których rejestrowanie narażenia było aktywne. Dlatego też funkcja rejestrowania powinna być stale aktywna."</string>
-    <!-- XHED: risk details - infection period logged information body, below behaviors -->
-    <string name="risk_details_information_body_period_logged_assessment">"Rejestrowanie narażenia obejmuje ostatnie 14 dni. W tym czasie funkcja rejestrowania w Twoim smartfonie była aktywna przez %1$s dni. Aplikacja automatycznie usuwa starsze dzienniki, ponieważ nie są one już istotne dla zapobiegania zakażeniom."</string>
+    <!-- XTXT: risk details - infection period logged information body, under 14 days -->
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/res/values-pl/strings.xml:347]: same as above

File: Corona-Warn-App/src/main/res/values-ro/strings.xml [hunk 4]
```diff
@@ -344,8 +344,8 @@
     <string name="risk_details_subtitle_period_logged">"Această perioadă este inclusă în calcul."</string>
     <!-- XHED: risk details - infection period logged information body, below behaviors -->
     <string name="risk_details_information_body_period_logged">"Riscul dvs. de infectare poate fi calculat doar pentru perioadele în care a fost activă înregistrarea în jurnal a expunerilor. Prin urmare, caracteristica de înregistrare în jurnal trebuie să rămână permanent activă."</string>
-    <!-- XHED: risk details - infection period logged information body, below behaviors -->
-    <string name="risk_details_information_body_period_logged_assessment">"Înregistrarea în jurnal a expunerilor acoperă ultimele 14 zile. În această perioadă, caracteristica de înregistrare în jurnal de pe smartphone-ul dvs. a fost activă timp de %1$s zile. Aplicația șterge automat înregistrările mai vechi din jurnal, întrucât acestea nu mai sunt relevante pentru prevenirea infectării."</string>
+    <!-- XTXT: risk details - infection period logged information body, under 14 days -->
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/res/values-ro/strings.xml:347]: same as above

File: Corona-Warn-App/src/main/res/values-tr/strings.xml [hunk 5]
```diff
@@ -344,8 +344,8 @@
     <string name="risk_details_subtitle_period_logged">"Bu dönem hesaplamaya dahil edildi."</string>
     <!-- XHED: risk details - infection period logged information body, below behaviors -->
     <string name="risk_details_information_body_period_logged">"Enfeksiyon riskiniz yalnızca maruz kalma günlüğünün etkin olduğu dönemler için hesaplanabilir. Bu nedenle günlüğe kaydetme özelliğinin sürekli etkin kalması gerekir."</string>
-    <!-- XHED: risk details - infection period logged information body, below behaviors -->
-    <string name="risk_details_information_body_period_logged_assessment">"Maruz kalma günlüğü son 14 günü kapsar. Bu süre boyunca akıllı telefonunuzdaki günlüğe kaydetme özelliği %1$s gün etkindi. Uygulama, enfeksiyondan korunma için artık ilgili olmadığından daha eski kayıtları otomatik olarak siler."</string>
+    <!-- XTXT: risk details - infection period logged information body, under 14 days -->
```
Comment (harambasicluka) [inline Corona-Warn-App/src/main/res/values-tr/strings.xml:347]: same as above

**Review by harambasicluka — CHANGES_REQUESTED** [review]
Please redo changes in all languages except german and default. This will prevent merge conflicts.

**Review by ralfgehrer — APPROVED** [review]

**Review by janetback — APPROVED** [review]
UA reviewed

**Review by harambasicluka — APPROVED** [review]

**Pull request merged**